src/components/Menu/index.js

```diff
@@ -115,6 +115,8 @@ function Menu({ links, children, socialLinks, ...props }) {
       {links.map((item, index) => (
         <Grid item key={item.label} className={classes.menu}>
           <Button
+            disableFocusRipple
+            disableRipple
```

Comment:
Since we don't want this on the whole site, just disable it in the [theme](https://material-ui.com/getting-started/faq/#how-can-i-disable-the-ripple-effect-globally)

Reply:
Moved to theme: https://github.com/CodeForAfrica/PesaYetu/blob/6d40e9289e2b8a946236c3a38f69d8914eee0dbc/src/theme/index.js#L101-L105